starfish/intensity_table/intensity_table.py

```diff
@@ -172,6 +173,9 @@ def save(self, filename: str) -> None:
             Name of Netcdf file
 
         """
+        if LOG in self.attrs:
+            # if log info, json encode it
+            self.attrs[LOG] = LogEncoder().encode(self.attrs[LOG])
```

Comment:
This is kind of odd because if you save it twice, the second time, you're going to be writing some weird stuff...